### PR TITLE
Feature: add a toggle fullscreen button

### LINKED ISSUE #25: Feature: add a toggle fullscreen button
### Feature

Users often need to switch to fullscreen mode for a better viewing experience. Adding a dedicated toggle fullsceen button will improve the user experience by allowing quick access to fullscreen mode.

### Additional details

- A button labeled 'Fullscreen' or with a fullscreen icon is to be added.
- Clicking the button toggles between fullscreen and normal mode.
- If it's a button, then it should show the current state (e.g., "Exit Fullscreen" when in fullscreen mode).
- It should be compatible with all the browsers.

### PR DESCRIPTION
**Pull request by aditya-bhaumik**
Fixes #25 

I fulfilled all the following requirements :

A button labeled 'Fullscreen' or with a fullscreen icon is to be added.
Clicking the button toggles between fullscreen and normal mode.
If it's a button, then it should show the current state (e.g., "Exit Fullscreen" when in fullscreen mode).
It should be compatible with all the browsers.

Apart from that i have added hover effect to the button it changes color when the pointer is on it 

Normal screen mode

image

Full screen mode 

image

Hover effect 
image

### CODE REVIEW

**Comment by sk66641** [comment]
@aditya-bhaumik there is an issue like when i switch to full screen mode using browser zoom option or F11 key, and then i try to exit full screen using button, that doesn't work
[Screencast from 29-05-24 01:32:18 AM IST.webm](https://github.com/sk66641/Random-Disco-Light-Simulator/assets/145834771/b51bc235-c6c4-4780-a043-f60d17f312ac)

File: style.css [hunk 1]
```diff
@@ -252,3 +252,21 @@ a:hover {
...
+    cursor: pointer;
+    border: none;
+    background-color: #00b3ff;
+    color: white;
+    border-radius: 5px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+}
+
+#fullscreenBtn:hover {
+    background-color: #71ff0c;
```
Comment (sk66641) [inline style.css:271]: @aditya-bhaumik Can you change this line 271 of style.css to, *background: linear-gradient(to right, #ff512f, #f09819);*

File: style.css [hunk 2]
```diff
@@ -252,3 +252,21 @@ a:hover {
...
+
+#fullscreenBtn {
+    position: fixed;
+    top: 20px;
+    left: 20px;
+    padding: 10px 20px;
+    font-size: 16px;
+    cursor: pointer;
+    border: none;
+    background-color: #00b3ff;
```
Comment (sk66641) [inline style.css:264]: @aditya-bhaumik Can you change this line 264 of style.css to,  *background: linear-gradient(#1845ad, #23a2f6);*

**Comment by aditya-bhaumik** [comment]
@sk66641 i have made the 2 changes in the code

**Comment by aditya-bhaumik** [comment]
@sk66641 please add the gssoc levels in this pull request also and assign me

**Pull request merged**